go.mod

```diff
@@ -87,3 +87,5 @@ replace github.com/openstack-k8s-operators/octavia-operator/api => ./api
 // mschuppert: map to latest commit from release-4.13 tag
 // must consistent within modules and service operators
 replace github.com/openshift/api => github.com/openshift/api v0.0.0-20230414143018-3367bc7e6ac7 //allow-merging
+
+replace github.com/openstack-k8s-operators/mariadb-operator/api => github.com/zzzeek/mariadb-operator/api v0.3.1-0.20240219194300-cc53ce6bfb27 //allow-merging
```

Comment:
Needed?

Reply:
we will have to remove this before merge.  right now https://github.com/openstack-k8s-operators/mariadb-operator/pull/184 has not merged yet, so your build here can't succeed without this replace at the moment.

Reply:
The linked PR has been merged now, so I'm removing this line.